core/ledger/util/couchdb/couchdb_test.go

```diff
@@ -631,7 +633,7 @@ func TestDBRequestTimeout(t *testing.T) {
 }
 
 func TestDBTimeoutConflictRetry(t *testing.T) {
-
+	startCouchDB()
```

Comment:
For tests that have their own CreateCouchInstance call, why do we need to start the DB instance?

Reply:
If I understand this correctly, **CreateCouchInstance** uses __config__ to start a couchDB instance, which inside __config__ contains address for couchDB container, the call itself does not start the container, only creates an instance. 
**startCouchDB** will start the external couchDB container if it has not been started by previous tests, and populate a global variable to store couchDB address which will be consumed by __config__.

Reply:
Ah, got it, thank you!